public/javascript/controllers/admin-ctrl.js

```diff
@@ -64,6 +75,9 @@ angular.module('harvestv2')
                 $scope.appLogsSearch  = $scope.appLogs = LogsFactory.query({lo_log_level: 'app_activity'});
             }
 
+            /*
+            This function filters the table of application logs based on the text in the search box - which is passed in the searchText variable
+             */
             $scope.filterAppLogs = function (searchText) {
```

Comment:
@nick22891 this should be temporary.  The filtering shouldn't occur on the front end of the application.  Imagine the search logs being over 10k records and fetching them each time is burdensome on the platform.  

The `LogsFactory` is fetching everything with no sorting - i believe the most current items are at the bottom of the list.